D.S Linkedlist/Linkedlist-hw2.php

```diff
@@ -0,0 +1,124 @@
+<?php
+class Node {
+   
+    private $data;
+    private $next; 
+
+    public function __construct($data = 0, $next = null)  // default value of $data is 0, $next is null
+    {
+        $this->data = $data; // initial $data
+        $this->next = $next; // initial $next
+    }
+
+    public function getData()
+    {
+        return $this->data;
+    }
+
+    public function setData($data)
+    {
+        $this->data = $data;
+    }
+
+    public function getNext()
+    {
+        return $this->next;
+    }
+
+    public function setNext($next)
+    {
+        $this->next = $next;
+    }
+
+    public function getPrev()
+    {
+        return $this->prev;
+    }
+    public function setPrev()
+    {
+        $this->prev = $prev;

```

Comment:
You were calling a property that does not exist in the class. Please double-check and resolve this issue. Thanks!

![image](https://user-images.githubusercontent.com/83938068/117909615-bf8f1b00-b304-11eb-939a-16c7ece7c0ab.png)

Reply:
I overlooked it. I'm sorry.　Please check the corrected version.